services/app-api/serverless.yml

```diff
@@ -20,24 +19,22 @@ plugins:
   - serverless-iam-helper
 
 custom:
-  stage: ${opt:stage, self:provider.stage}
   region: ${opt:region, self:provider.region}
-  appVersion: ${env:APP_VERSION}
-  applicationEndpoint: ${cf:ui-${self:custom.stage}.CloudFrontEndpointUrl, env:APPLICATION_ENDPOINT}
-  appApiGatewayId: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:infra-api-${self:custom.stage}.ApiGatewayRestApiId}
-  appApiGatewayRootResourceId: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:infra-api-${self:custom.stage}.AppApiGatewayRootResourceId}
+  applicationEndpoint: ${cf:ui-${sls:stage}.CloudFrontEndpointUrl, env:APPLICATION_ENDPOINT}
+  appApiGatewayId: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:infra-api-${sls:stage}.ApiGatewayRestApiId}
+  appApiGatewayRootResourceId: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:infra-api-${sls:stage}.AppApiGatewayRootResourceId}
   emailSource: ${env:SES_SOURCE_EMAIL_ADDRESS}
   emailerMode: ${env:EMAILER_MODE}
   emailReviewTeamAddresses: ${env:SES_REVIEW_TEAM_EMAIL_ADDRESSES}
   emailRatesTeamAddresses: ${env:SES_RATES_EMAIL_ADDRESSES}
   emailReviewHelpAddress: ${env:SES_REVIEW_HELP_EMAIL_ADDRESS}
   emailRateHelpAddress: ${env:SES_RATE_HELP_EMAIL_ADDRESS}
   emailDevTeamHelpAddress: ${env:SES_DEV_TEAM_HELP_EMAIL_ADDRESS}
-  auroraArn: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:postgres-${self:custom.stage}.PostgresAuroraArn }
+  auroraArn: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:postgres-${sls:stage}.PostgresAuroraArn }
   iamPath: ${env:IAM_PATH, "/"}
   iamPermissionsBoundaryPolicy: ${env:FULL_IAM_PERMISSIONS_BOUNDARY_POLICY, ""}
-  document_uploads_bucket_arn: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:uploads-${self:custom.stage}.DocumentUploadsBucketArn}
-  secretsManagerSecret: aurora_postgres_${self:custom.stage} #pragma: allowlist secret
+  document_uploads_bucket_arn: ${env:CF_CONFIG_IGNORED_LOCALLY, cf:uploads-${sls:stage}.DocumentUploadsBucketArn}
+  secretsManagerSecret: aurora_postgres_${sls:stage} #pragma: allowlist secret
```

Comment:
what is the “sis” directive here?

Reply:
`sls` is just how serverless talks about itself. There's an `sls` command that is just a symlink to `serverless` as well for CLI commands. Not sure why they decided to do that.